tests/plugins/elementspath/manual/elementspath.html

```diff
@@ -0,0 +1,82 @@
+<textarea id="editor1">
```

Comment:
Instead of duplicating the same content in both editors you may put part of it with some elements (e.g. image, styling, widget) in the first one and rest (e.g. tables, list) in the second one. It is always less monotonic to test different editor with different contents (ofc not applicable in all cases, but here it will be ok).